app-emulation/libvirt-glib/libvirt-glib-4.0.0.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit meson vala
+
+DESCRIPTION="GLib and GObject mappings for libvirt"
+HOMEPAGE="https://libvirt.org/ https://gitlab.com/libvirt/libvirt-glib/"
+SRC_URI="https://libvirt.org/sources/glib/${P}.tar.xz"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~ppc64 ~x86"
+IUSE="+introspection +vala"
+REQUIRED_USE="vala? ( introspection )"
+
+BDEPEND="
+        dev-util/glib-utils
+        dev-util/gtk-doc-am
+        >=dev-util/intltool-0.35.0
+        virtual/pkgconfig
+        vala? ( $(vala_depend) )"
+RDEPEND="
+        dev-libs/libxml2:2
+        >=app-emulation/libvirt-1.2.6:=
+        >=dev-libs/glib-2.38.0:2
+        introspection? ( >=dev-libs/gobject-introspection-1.36.0:= )"
+
+DEPEND="${BDEPEND}
+        ${RDEPEND}"
+
+src_prepare() {
+        default
+        use vala && vala_src_prepare
+}
+
+src_configure() {
+        local emesonargs=(
+                $(meson_feature introspection)
+                $(meson_feature vala vapi)
```

Comment:
`meson_options.txt` has two other configuration options that I think we should have:
````
option('docs', type: 'feature', value: 'auto', description: 'whether to generate documentation')
option('tests', type: 'feature', value: 'auto', description: 'whether to build tests')
````

`docs` should be controlled by an `IUSE=gtk_doc` flag and should depend on `dev-util/gtk-doc`.

`tests` should be controlled by an `IUSE=test` flag that is implicitly enabled when building with `FEATURES=test`.